ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/tracing/vm/VmTraceGenerator.java

```diff
@@ -101,6 +98,9 @@ private boolean mustIgnore(final TraceFrame frame) {
     if ("STOP".equals(frame.getOpcode()) && transactionTrace.getTraceFrames().size() == 1) {
```

Comment:
Style... could these be if (...) { return ... } else if (...) { return ... } else { return ...} - having the returns at the same indent and in an else-if chain I find increases readability.  It makes the effect explicit, if X then A, if Y then B, otherwise C.

Reply:
Sure. Fixed.